qiskit/visualization/counts_visualization.py

```diff
@@ -70,13 +70,13 @@ def _is_deprecated_data_format(data) -> bool:
 )
 def plot_histogram(
     data,
-    figsize=(7, 5),
+    figsize=None,
     color=None,
     number_to_keep=None,
     sort="asc",
     target_string=None,
     legend=None,
-    bar_labels=True,
+    bar_labels=False,
```

Comment:
was this change intentional?

Reply:
Ahh, this was because the labels are redundant given the y-axis labels, and easily becomes messy when the number of bit-strings is large, and the figure dims are not set properly (if possible).

Reply:
is the change in the default worthy an `upgrade` note in the release note?

Reply:
Undoing this change, as it is the only blocker. For sure, it can be done in 2.0 as a breaking change. 

```suggestion
    bar_labels=True,
```